CRM/Sqltasks/Form/SqltasksExecutionList.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+use CRM_Sqltasks_ExtensionUtil as E;
+
+class CRM_Sqltasks_Form_SqltasksExecutionList extends CRM_Core_Form {
```

Comment:
We need a paginator for this as we run tasks very often. We may also be able to ship a packaged searchkit search for this instead.